src/components/Molecule/Task/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+import StyledCheckBoxWrapper from "../../Atoms/CheckBox/index";
+import Input from "../../Atoms/Input/index.jsx";
+import StyledEditButtonWrapper from "../../Atoms/EditButton/index";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+
+const TaskSample = ({ defaultValue, defaultIsEditing, OnTaskChange }) => {

```

Comment:
story.jsxの方と比較するとスペルミスがあるみたいなのでエラーが起こっていました
```suggestion
const TaskSample = ({ defaultValue, defaultIsEditing, onTaskChange }) => {
```